tests/test_h5store.py

```diff
@@ -661,13 +664,16 @@ def test_single_writer_multiple_reader_same_process(self):
                         self.assertEqual(reader1['test'], True)
                         self.assertEqual(reader2['test'], True)
 
+    @unittest.skipIf(WINDOWS, 'This test fails for an unknown reason on Windows.')
```

Comment:
I am not too worried about this, but it would be nice to know why the tests fails.

Reply:
I had to cut my losses, I couldn't figure it out after ~1 hour. I'm fine with skipping it.

Reply:
Makes sense. Ultimately unix-like systems dominate, so this shouldn't be too big an issue.